src/main/java/sirius/biz/codelists/CodeLists.java

```diff
@@ -438,7 +438,21 @@ public String getValue(@Nonnull String codeListName, @Nullable String code, @Nul
      */
     @Nullable
     public String getTranslatedValue(@Nonnull String codeListName, @Nullable String code) {
-        String value = getValue(codeListName, code, NLS.getCurrentLang());
+        return getTranslatedValue(codeListName, code, NLS.getCurrentLang());
+    }
+
+    /**
+     * Returns the value translated from the given code list associated with the given code and language code.
+     * <p>
+     * If no matching entry exists, the code itself will be returned.
+     *
+     * @param codeListName the code list to search in
+     * @param code         the code to lookup
+     * @param langCode     the language code to translate into
+     * @return the translated value associated with the code or the code itself if no value exists
+     */
+    public String getTranslatedValue(@Nonnull String codeListName, @Nullable String code, String langCode) {
+        String value = getValue(codeListName, code, langCode);
         if (value != null && value.startsWith("$")) {
             return Value.of(value).translate().getString();
```

Comment:
Don't we need the langcode for Value.of(value).translate() too?

Reply:
Yes we do actually need that code in that method. Added it.